content/02.main-text.md

```diff
@@ -121,6 +121,7 @@ All citations point to a standard identifier, for which Manubot automatically re
 Table @tbl:citations presents the supported identifiers and example citations before and after Manubot processing.
 Authors can optionally define citation tags to provide short readable alternatives to the citation identifiers.
 Metadata is exported to [Citation Style Language (CSL) JSON Items](http://citeproc-js.readthedocs.io/en/latest/csl-json/markup.html#items), an open standard that is widely supported by reference managers [@doi:10.1007/978-3-319-00026-8_8; @doi:10.1080/02763869.2012.641841].
+Manubot ensures its CSL Items validate according to the official JSON schema by removing problematic fields, which are prone to crashing downstream citation processors, such as [pandoc-citeproc](http://hackage.haskell.org/package/pandoc-citeproc) in Manubot's case.
```

Comment:
I accidentally dropped a comment from my review

Can we restate this to emphasize the most important parts for a general audience?
> Manubot ensures its CSL Items validate and removes problematic fields, which are prone to crashing downstream citation processors.